src/custom/components/AccountDetails/Copy.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import useCopyClipboard from '@src/hooks/useCopyClipboard'
```

Comment:
We prefer to use the `@src` reference only when required.
This should work with the default path.

I know this import comes automatically from VSCode, just keep an eye out :)

```suggestion
import useCopyClipboard from 'hooks/useCopyClipboard'
```